### ISSUE TITLE
Can I trade on this?

### ISSUE THREAD

**Issue by tinyz666** [calc84maniac/tiboyce#99]
Has trading on this emulator been made yet? if so, do I use a cable

**Comment by ndye** [comment]
No, it is one of the future features that is on the list of things to do see #30.
